go/tools/gopackagesdriver/bazel_json_builder.go

```diff
@@ -49,6 +51,29 @@ func (b *BazelJSONBuilder) fileQuery(filename string) string {
 		label = fmt.Sprintf("@%s//%s", matches[1], strings.Join(matches[2:], ":"))
 	}
 
+	relToBin, err := filepath.Rel(b.bazel.info["bazel-bin"], filename)
```

Comment:
Since rules_go relies so heavily on transitions, I would expect the top-level configuration to differ from an individual generated file's configuration rather often. What do you think of making this relative to `bazel-out` instead and then dropping the first two segments (of the form `k8-opt/bin`)?

Reply:
I wasn't aware that these could differ significantly. I'll modify it.